updatesnap/updatesnapyaml.py

```diff
@@ -107,8 +107,8 @@ def main():
         has_update = True
 
     if has_update:
-        print(manager_yaml.get_yaml())
-        sys.exit(0)
+        with open('temp-file', 'w') as output_file:
```

Comment:
This sends the output to "temp-file", while the action expects "output_file".

Reply:
oh oops. An artifact from a previous suggestion.